### ISSUE TITLE
Bowtie overcommits CPU ressources

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/tools-iuc#5983]
`bowtie` seems to be extremely efficient, i.e. `X` `GALAXY_SLOTS` are used the CPU usage is most of the time larger than `X`*100% (if checked in `top`). This seems to be because the `-p` parameter seems to control the number of (additional) alignment threads (probably plus 1 control thread). 

Additionally we pipe the result to `samtools` using potentially  `GALAXY_SLOTS`  more slots (in practice most of the time less).

In summary I would suggest to remove the pipes (to the cost of having an additional temporary file.)
